### DISCUSSION TITLE
can anyone with 5950X reach the speed in "MadMAx43v3r Plotter" google sheet that declare a 16.13 min?

### DISCUSSION THREAD

**Discussion by ruixingchen** [madMAx43v3r/chia-plotter#999] [Show and tell]
"ubuntu 21.04 dinamic oc ccx1 4775 ccx2 4625 1.275v XFS 2048 filesystem  https://i.imgur.com/WLMusTv.png"

that man said with a 32gb RAM and PCIE 4.0 disk he can reach 16.1333 min, can any one reach that? I tried DDR4 3600 and pm9a3 on 5950X, the result is not good, still 30 min

**Comment by hajes** [comment]
I am getting ~17.5min with 5950X @ 4.3GHz + 128GB 3600 MHz + Corsair MP600 2TB in Clear Linux...around 55W/plot, 80+ plots/day

If he is running some enterprise grade SSD...it may be possible.
